src/main/java/com/ft/methodearticleinternalcomponentsmapper/transformation/BlockquoteXMLEventHandler.java

```diff
@@ -0,0 +1,52 @@
+package com.ft.methodearticleinternalcomponentsmapper.transformation;
+
+import com.ft.bodyprocessing.BodyProcessingContext;
+import com.ft.bodyprocessing.writer.BodyWriter;
+import com.ft.bodyprocessing.xml.eventhandlers.BaseXMLEventHandler;
+import org.apache.commons.lang.StringUtils;
+
+import javax.xml.stream.XMLEventReader;
+import javax.xml.stream.XMLStreamException;
+import javax.xml.stream.events.StartElement;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+public class BlockquoteXMLEventHandler extends BaseXMLEventHandler {
+
+    private static final String BLOCKQUOTE_TAG = "blockquote";
+    private static final String PARAGRAPH_TAG = "p";
+    private static final String CITE_TAG = "cite";
+
+    private BlockquoteXMLParser blockquoteXMLParser;
+
+    public BlockquoteXMLEventHandler(BlockquoteXMLParser blockquoteXMLParser) {
+        this.blockquoteXMLParser = blockquoteXMLParser;
+    }
+
+    @Override
+    public void handleStartElementEvent(StartElement event, XMLEventReader xmlEventReader, BodyWriter eventWriter, BodyProcessingContext bodyProcessingContext) throws XMLStreamException {
+        BlockquoteData blockquoteData = blockquoteXMLParser.parseElementData(event, xmlEventReader, bodyProcessingContext);
+
+        List<String> paragraphs = blockquoteData.getParagraphs();
+        if (!paragraphs.isEmpty()) {
```

Comment:
It could be helpful to log if there is no "p" tags, as you don't write the blockquote at all in this case (which is good as it is an invalid blockquote), so we can easily find this issue if it happens.